include/modules/dummy/coin-acceptor.h

```diff
@@ -0,0 +1,37 @@
+/*
+	Copyright (C) 2020 Samotari (Charles Hill, Carlos Garcia Ortiz)
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef BLESKOMAT_MODULES_DUMMY_COIN_ACCEPTOR_H
+#define BLESKOMAT_MODULES_DUMMY_COIN_ACCEPTOR_H
+
+#include "logger.h"
+
+#include <string>
+
+namespace coinAcceptor {
+	void init();
+	void loop();
+	float getAccumulatedValue();
+	float getMaxCoinValue();
+	void reset();
+	bool isOff();
+	bool isOn();
+	void on();
+	void off();
+}
+
+#endif
```

Comment:
I don't think the dummy/coin-acceptor files are used anywhere? We use the dummy/screen because it's a necessary module. In main, there are if statements to check for the COIN_ACCEPTOR flag and only use the `coinAcceptor` namespace if the flag is set.

Reply:
You are right, removed!